packages/bits/demo/src/components/demo/common/unit-conversion-service/unit-conversion-service-separate-unit-display/unit-conversion-service-separate-unit-display.example.component.ts

```diff
@@ -21,7 +21,10 @@ export class UnitConversionServiceSeparateUnitDisplayExampleComponent implements
         this.num = num;
         const conversion: IUnitConversionResult = this.unitConversionService.convert(this.num, UnitBase.Standard, 2);
         this.unitDisplay = this.unitConversionService.getUnitDisplay(conversion, "hertz");
+        // An undefined getUnitDisplay return value indicates the input value was too large to be converted,
+        // so the base unit and scientific notation can be used as fallbacks.
         if(this.unitDisplay) {
+            this.unitDisplay = this.unitConversionService.getUnitDisplayBaseValue("hertz");
```

Comment:
Oh, I think the comment and unitDisplay fallback go in the else block rather than this one.